src/flynt/api.py

```diff
@@ -105,7 +115,13 @@ def default_result():
     return True, changes, len(contents), len(new_code)
 
 
-def fstringify_files(files, multiline, len_limit, transform_concat):
+def fstringify_files(
```

Comment:
there are so-called integration tests that invoke file processing, can you add a test that without the `transform_join` flag there is no change? I think in current files there are just no candidate expressions, so they might not be able to tell how is software behaving in that regard..

Reply:
Sure, but maybe we should get #146 merged first to avoid double work in that regard?

Reply:
Rebased + added integration test that checks that a candidate join doesn't get changed by default.